hyperliquid/exchange.py

```diff
@@ -49,25 +49,28 @@ def _post_action(self, action, signature, nonce):
         return self.post("/exchange", payload)
 
     def order(
-        self, coin: str, is_buy: bool, sz: float, limit_px: float, order_type: OrderType, reduce_only: bool = False
+        self, coin: str, is_buy: bool, orders: List[Dict[str, float]], order_type: OrderType, reduce_only: bool = False
```

Comment:
For the batch api, can we let people specify coin, side, order_type and reduce_only for every order, not just sz and limit_px? I think it's good to have this API be maximally flexible